### ISSUE TITLE
double survey messes the comments

### ISSUE THREAD

**Issue by rdemaria** [MethodicalAcceleratorDesign/MAD-X#787]
```
option, -echo, warn, info;
m1:marker,comments=fafdadfaf;
ss: sequence,l=10;
m1,at=4;
endsequence;
beam;
use,sequence=ss;survey; survey;
show,m1;
```

```

  ++++++++++++++++++++++++++++++++++++++++++++
  +     MAD-X 5.05.01  (64 bit, Linux)       +
  + Support: mad@cern.ch, http://cern.ch/mad +
  + Release   date: 2019.06.07               +
  + Execution date: 2019.06.19 19:19:57      +
  ++++++++++++++++++++++++++++++++++++++++++++
option, -echo, warn, info;

+++ dumping element m1                  parent marker            
command: m1  module: element
....
parameter: comments   string: ss$start:1

  Number of warnings: 0

  ++++++++++++++++++++++++++++++++++++++++++++
  +          MAD-X finished normally         +
  ++++++++++++++++++++++++++++++++++++++++++++

```

**Comment by tpersson** [comment]
Just ran the example and can see the problem and I also notice that the same thing does not happen when doing two TWISS. I will have a look inside the code now to see what the problem might be.

**Comment by madcern** [comment]
Thanks for reporting this and also the clear minimalistic example showing the issue! It made it much easier to debug. The bug is fixed in #788 .
